monero-outreach-docs/translations/fr/archive/dandelion_fr.md

```diff
@@ -0,0 +1,67 @@
+
+### Le Réseau P2P de monero
+
+Monero possède deux éléments conceptuels: 1) un réseau P2P et 2) des applications qui s'exécutent sur ce réseau. Les applications gèrent les adresses, les clés et les transactions, tandis que le réseau organise l'information et s'assure de la transmission des données. Pour bien comprendre Dandelion++, il est pertinent de considérer la structure du réseau P2P ainsi que la manière dont les portefeuilles s'en servent pour communiquer les transactions.
+
+Sa structure comprend des milliers de nœuds répartis de par la planète — des dizaines de milliers si on compte les nœuds passifs — chacun connecté, via Internet, habituellement à un petit groupe d'autres nœuds appelés pairs [1]. Les nœuds communiquent avec leurs pairs en utilisant le même protocole TCP ("Transmission Control Protocol" - Protocole de Contrôle de Transmission) utilisé par les fureteurs et les serveurs Web. Les pairs d'un nœud paraissent aléatoires et n'ont pas besoin de rapprochement géographique. Un nombre commun de pairs est huit, mais certains nœuds en ont des centaines. Chaque nœud trouve ses pairs en envoyant d'abord une requête à un nœud maître pour sa liste de pairs, pour ensuite envoyer la même requête aux pairs de cette liste, et ainsi de suite. Le résultat est un réseau de plusieurs ordinateurs, chacun communicant par TCP avec au mieux une visibilité partielle et limitée du réseau entier.
+
+Ces nœuds agissent comme un portail pour permettre au portefeuilles de communiquer avec le réseau P2P. Certains portefeuilles, comme le portefeuille GUI officiel, peuvent exécuter leur propre nœud, tandis que d'autres, comme MyMonero sur un téléphone intelligent, utilisent toujours un nœud distant. Les portefeuilles dépensent Monero en créant et diffusant une transaction via son nœud portail, afin que la transaction trouve ultimement son chemin vers le nœud d'un mineur pour être incluse dans un bloc miné. Chaque transaction comporte une quantité limitée d'informations, mais qui contient malgré tout des données significatives, incluant un ensemble de sources possibles des fonds dépensés.
```

Comment:
téléphone intelligent >>> téléphone intelligent (smartphone) 

Reason: The literal translation may surprise some French speakers of specific regions.

Reply:
Good point, Integrated.